scripts/ci/test_ref_doc.sh

```diff
@@ -17,4 +19,7 @@ echo "Installed."
 
 cd doc/sphinx; make xml
```

Comment:
Why is this script using make xml directly? It should be using azhelpgen since that was where the problem was. As nice as this is, by calling `make xml` directly, even this would not have prevented the docs regression.

Reply:
This was the previous behavior of the script. `make xml` and the azhelpgen main script both use `sphinx-build` to build the xml doc files. Their behavior is identical. I am open to using either the make file or the python main method that calls `sphinx-build`

Reply:
Use azhelpgen because that is what the docs team's CI is using.